src/wrench/models.py

```diff
@@ -35,7 +35,7 @@ def __hash__(self):
 
 
 class Resource(IdEqualityMixin,
-               namedtuple('Resource', 'id name uri description username secret encrypted_secret tags')):
```

Comment:
I think `encrypted_secret` is still needed because it's used when adding resources. If you search for `encrypted_secret` you'll see this field is used in several places in the code.